src/Microsoft.Azure.WebJobs.Host/Bindings/BindingProviders/ExactTypeBindingProvider.cs

```diff
@@ -8,23 +8,28 @@
 
 namespace Microsoft.Azure.WebJobs.Host.Bindings
 {
-    // BindingProvider to bind to an exact type. Useful for binding to a client object. 
-    internal class ExactTypeBindingProvider<TAttribute, TUserType> : IBindingProvider
+    // BindingProvider to bind to type. It can user Converter manager to convert from TMiddleType to the user
```

Comment:
removed this file. 

---
In reply to: [96106019](https://github.com/Azure/azure-webjobs-sdk/pull/967#discussion_r96106019) [](ancestors = 96106019)